opensrp-chw-anc/src/main/java/org/smartregister/chw/anc/util/DBConstants.java

```diff
@@ -34,6 +34,7 @@
         public static final String DATE_CREATED = "date_created";
         public static final String PHONE_NUMBER = "phone_number";
         public static final String HAS_ANC_CARD = "has_anc_card";
+        public static final String DELIVERY_KIT = "delivery_kiy";
```

Comment:
did you mean delivery_kit?

Reply:
Yes its delivery_kit, spell corrected now.